x-pack/plugins/apm/public/components/app/mobile/search_bar.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { QueryDslQueryContainer } from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
+import {
+  EuiFlexGroup,
+  EuiFlexGroupProps,
+  EuiFlexItem,
+  EuiSpacer,
+} from '@elastic/eui';
+import React from 'react';
+import { useBreakpoints } from '../../../hooks/use_breakpoints';
+import { ApmDatePicker } from '../../shared/date_picker/apm_date_picker';
+import { KueryBar } from '../../shared/kuery_bar';
+import { TimeComparison } from '../../shared/time_comparison';
+import { TransactionTypeSelect } from '../../shared/transaction_type_select';
+import { MobileFilters } from './service_overview/filters';
+
+interface Props {
+  hidden?: boolean;
+  showKueryBar?: boolean;
+  showTimeComparison?: boolean;
+  showTransactionTypeSelector?: boolean;
+  showMobileFilters?: boolean;
+  kueryBarPlaceholder?: string;
+  kueryBarBoolFilter?: QueryDslQueryContainer[];
+}
+
+export function MobileSearchBar({
```

Comment:
Is it possible to create the `MobileSearchBar` as an extension of `SearchBar`? From what I see this one it's pretty similar to `SearchBar` except for the last part. Maybe something like could work?
```
export function MobileSearchBar(){
...
return (
  <EuiFlexGroup>
      <SearchBar />
      <EuiFlexItem style={{ minWidth: 300 }}>
           <MobileFilters />
       </EuiFlexItem>
  </EuiFlexGroup>
)};
```

Reply:
The layout is slightly different for the mobile according to the [mockups](https://www.figma.com/file/8tTzdCtAgGfmbCciZEq9La/Mobile-APM?node-id=97%3A164583&t=AGJ4Ob6aUGyWetLk-0) that's why i created a separate component

mobile
<img width="1329" alt="image" src="https://user-images.githubusercontent.com/3369346/208871055-a8e5e918-244b-4e62-93dc-bf53f4f26b4b.png">
vs
<img width="1536" alt="Screenshot 2022-12-21 at 10 25 23" src="https://user-images.githubusercontent.com/3369346/208871089-f0e056d8-b1ca-48a3-b1eb-c495e8511eba.png">

Let me know if you could think of a better approach for this

Reply:
@boriskirov what do you think about unifying those layouts?